rusk/src/lib/services/blindbid/score_gen_handler.rs

```diff
@@ -110,11 +110,12 @@ fn parse_score_gen_params(
 // desired inputs.
 fn gen_blindbid_proof(circuit: &mut BlindBidCircuit) -> Result<Proof> {
     // Read ProverKey of the circuit.
-    let prover_key = rusk_profile::keys_for("dusk-blindbid")
-        .get_prover("blindbid")
-        .expect("Failed to get blindbid circuit keys from rusk_profile.");
+    let pk =
+        rusk_profile::keys_for(&BlindBidCircuit::CIRCUIT_ID)?.get_prover()?;
 
-    let prover_key = ProverKey::from_bytes(&prover_key[..])?;
+    let prover_key = ProverKey::from_slice(&pk)?;
     // Generate a proof using the circuit
-    circuit.gen_proof(&crate::PUB_PARAMS, &prover_key, b"BlindBid")
+    circuit
+        .gen_proof(&crate::PUB_PARAMS, &prover_key, b"BlindBidProof")
+        .map_err(|e| anyhow::anyhow!("{:?}", e))
```

Comment:
I believe anyhow is being systematically removed from the stack

Reply:
Will do it in a follow up PR. Now BlindBid service is not even used nor tested. So I'm not so concerned about it.